### DISCUSSION TITLE
Why when we click fork on GitHub, it doesn't fork issues, wiki?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62332] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, Why when we click fork on GitHub, it doesn't fork issues, wiki?

**Comment by Mittelblut9** [comment, marked as answer]
You only fork the Repository. Issues and Pull Requests will stay on the original repository.

**Comment by h7x4** [comment]
Most forks are created in order for people to have a place to create branches and commits which will eventually become PRs. It doesn't make sense to copy all the issues over to these kinds of forks. The original issues and PRs weren't created in order for someone to solve and merge in their personal fork. They were targeted at the original project.

There are some kinds of forks where you're trying to maintain a new version of the project, and where the codebase tends to deviate from the original over time. You could make an argument that these kinds of forks would benefit from at least copying some issues over. But these kinds of forks are an edge case, and I don't think forks should be considered to be this kind by default.
